### ISSUE TITLE
Provide PDB's for EXE/DLL files of the browser (mainly for XP build)

### ISSUE THREAD

**Issue by IDA-RE-things** [Alex313031/thorium-legacy#76]
Hello, @Alex313031 ,
Can you provide also PDB files, to be able to debug separete problems with this browser ?
Same PDB's, as Supemium provides now on my request.
Mainly for XP 32-bit build.

**Comment by Alex313031** [comment]
@IDA-RE-things The PDB files would not really be of use. As part of the compiler optimizations of Thorium, stuff is inlined, and all debugging constructs are removed. In addition, when linking it uses /DEBUG:NONE, instead of the normal /DEBUG which is used on all build of Chromium (normally).

This makes the .pdbs very small, and doesn't really have enough information to do any proper debugging.

The latest progwrp should fix the handle leaks. If there are any other major issues that you notice, I could make a debug build and share it with you here along with the PDB, so you can inspect it.

**Comment by IDA-RE-things** [comment]
no, not for progwrp.dll problems. But for problems at all. For example crashes. Or hangs. Or performance issues.
Yes, special debug version with PDB's will help in this case I think. Except performance issues, where all should be optimized.
